### DISCUSSION TITLE
How to Run a Mainnet Hedera Mirror Node

### DISCUSSION THREAD

**Discussion by NehaSunkum** [hiero-ledger/hiero-mirror-node#7014] [Help] [answered]
It looks like the setup in this repository is geared towards testnet. How would I go about setting up a mainnet Hedera mirror node?

**Comment by steven-sheehy** [comment, marked as answer]
It's just that the default in the code is testnet. Please see our [docs](https://docs.hedera.com/hedera/core-concepts/mirror-nodes/run-your-own-beta-mirror-node/run-your-own-mirror-node-s3) in how to setup a mirror node. You would need GCS or S3 credentials and set the network to `MAINNET`.
